validator-api/src/config/mod.rs

```diff
@@ -2,18 +2,17 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use crate::config::template::config_template;
+use coconut_rs::{Base58, KeyPair};
 use config::NymConfig;
 use serde::{Deserialize, Serialize};
 use std::path::{Path, PathBuf};
 use std::time::Duration;
 
 mod template;
 
-const DEFAULT_VALIDATOR_REST_ENDPOINTS: &[&str] = &[
-    "http://testnet-finney-validator.nymtech.net:1317",
-    "http://testnet-finney-validator2.nymtech.net:1317",
-];
-const DEFAULT_MIXNET_CONTRACT: &str = "hal1k0jntykt7e4g3y88ltc60czgjuqdy4c9c6gv94";
+const DEFAULT_VALIDATOR_REST_ENDPOINTS: &[&str] =
+    &["http://testnet-milhon-validator1.nymtech.net:1317"];
+const DEFAULT_MIXNET_CONTRACT: &str = "punk18vd8fpwxzck93qlwghaj6arh4p7c5n89m7r3lj";
```

Comment:
That's definitely incorrect. Our current contract is `punk10pyejy66429refv3g35g2t7am0was7yalwrzen`